src/cl/lighthouse/lighthouse_launcher.star

```diff
@@ -353,46 +353,58 @@ def get_beacon_config(
         "--metrics-allow-origin=*",
         "--metrics-port={0}".format(BEACON_METRICS_PORT_NUM),
         # ^^^^^^^^^^^^^^^^^^^ METRICS CONFIG ^^^^^^^^^^^^^^^^^^^^^
+        # "--allow-insecure-genesis-sync",
     ]
 
     if network not in constants.PUBLIC_NETWORKS:
         cmd.append("--testnet-dir=" + constants.GENESIS_CONFIG_MOUNT_PATH_ON_CONTAINER)
-    else:
-        cmd.append("--network=" + network)
-        cmd.append("--checkpoint-sync-url=" + constants.CHECKPOINT_SYNC_URL[network])
-
-    if network == "kurtosis":
-        if boot_cl_client_ctxs != None:
+        if network == "kurtosis":  # Kurtosis
```

Comment:
i think we should take out `kurtosis` as a constant we have comparision at a lot of places

Reply:
I don't see much value in that :D

Reply:
if for some reason we rename kurtosis to `Kurtosis` you suddenly have to make the replacements throughout the repository and your risk running into typos. You might already have typo'd in some clients and not others and cann't know.

Reply:
https://github.com/kurtosis-tech/ethereum-package/pull/459/commits/906932d434d15e5ee0457d471af9a9e2c6e5c238